Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
```

Comment:
Gppd addition that is also consistant with the style of the rest of the code!

Reply:
I agree! I like the consistent styling convention and clarity of function names and variables